conf/accumulo-testing.properties

```diff
@@ -46,10 +46,10 @@ test.ci.common.auths=
 # Accumulo tserver properties to set when creating a table
 test.ci.common.accumulo.server.props=\
 compaction.service.cs1.planner=org.apache.accumulo.core.spi.compaction.DefaultCompactionPlanner \
-compaction.service.cs1.planner.opts.executors=\
-[{"name":"small","type":"internal","maxSize":"16M","numThreads":8},\
-{"name":"medium","type":"internal","maxSize":"128M","numThreads":4},\
-{"name":"large","type":"internal","numThreads":2}]
+compaction.service.cs1.planner.opts.groups=\
+[{"name":"small","maxSize":"16M"},\
+{"name":"medium","maxSize":"128M"},\
+{"name":"large"}]
```

Comment:
Will these names work? They don't match the names in the compactor config at https://github.com/apache/accumulo-testing/blob/elasticity/contrib/terraform-testing-infrastructure/modules/config-files/templates/cluster.yaml.tftpl#L41

Reply:
No you're right. They will need to be modified to match the resource groups that are created.